Resources/Private/Templates/CompactView/Index.html

```diff
@@ -1,19 +1,17 @@
-<link rel="stylesheet" type="text/css"
```

Comment:
CSS isn't needed anymore?

Reply:
I think this is handled by the JS now. I followed documentation of https://developer-docs.bynder.com/UI%20components/#compact-view where it only mentioned the <script> to setup.